src/py21cmfast/src/UsefulFunctions.c

```diff
@@ -716,7 +716,7 @@ float ComputeTau(struct UserParams *user_params, struct CosmoParams *cosmo_param
     return tau;
 }
 
-
+//TODO: THESE NEED UPDATING
```

Comment:
OK, let's update it :-)

Reply:
Apart from testing, what is the point of these functions? After all, we have much better parameter printing in the wrapper.

Reply:
I think the original idea was to be able to print the structs from C-code for debugging -- checking that the parameters all got passed through to C OK from the python. Since adding them we've added better logging in the python, so maybe it's not so important anymore, but I can imagine situations in which you want to debug this and then you'd have to rewrite the function if we remove it now.